tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -36,6 +37,18 @@ public function testFix($expected, $input = null, array $configuration = null)
         if (null !== $configuration) {
             $this->fixer->configure($configuration);
         }
+        $indent = '    ';
+        if (strpos((string) $input, "\t") !== false) {
+            $indent = "\t";
+        }
+        if (preg_match('/\n  \S/', (string) $input)) {
+            $indent = '  ';
+        }
+        $lineEnding = strpos((string) $input, "\r") === false ? "\n" : "\r\n";
```

Comment:
I retrieved your branch and rewrote the tests:
* removed the `\r` from the `test violation after opening parenthesis, with \r\n line endings` case
* added a new test that uses the same data provider and replaces all `\n` with `\r\n`

```php
/**
 * @dataProvider testFixProvider
 */
public function testFix(/* ... */)
{
    // current test without the \r check and always using \n as line ending
}

/**
 * @dataProvider testFixProvider
 */
public function testFixWithDifferentLineEndings(/* ... */)
{
    // ...
    $expected = str_replace("\n", "\r\n", $expected);
    if (null !== $input) {
        $input = str_replace("\n", "\r\n", $input);
    }

    $this->fixer->setWhitespacesConfig(new WhitespacesFixerConfig(
        $indent,
        "\r\n"
    ));

    $this->doTest($expected, $input);
}
```

This way, all cases are tested with `\n` **and** `\r\n`. I then got one failure, didn't investigate it for now, could you please have a look?